### PR TITLE
Update Kali Installation instructions

### PR DESCRIPTION
**Pull request by yeyintminthuhtut**
## PR Summary

Updated on powershell installation on kali on following.

1. libicu55 couldn't install with apt-get. So, provided url to download to install with dpkg.
2. updated libssl version is given url. 

## PR Checklist

Note: Please mark anything not applicable to this PR `NA`.

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [x] Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] User facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [ ] Issue filed - Issue link:
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
    - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.

### CODE REVIEW

**Comment by TravisEz13** [comment]
restarted linux ci due to bootstrap failure

**Comment by TravisEz13** [comment]
Your markdown has syntax errors.  See the errors in the CI system here:  https://ci.appveyor.com/project/PowerShell/powershell/build/v6.1.0-preview.7630#L83

File: docs/installation/linux.md [hunk 1]
```diff
@@ -574,8 +574,11 @@ and is a single binary.
 ```sh
 # Install prerequisites
 apt-get install libunwind8 libicu55
-wget http://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.0.0_1.0.1t-1+deb8u6_amd64.deb
-dpkg -i libssl1.0.0_1.0.1t-1+deb8u6_amd64.deb
+```
+(If you get an error in the next step when installing libicu55 using apt-get, download it from [here](http://archive.ubuntu.com/ubuntu/pool/main/i/icu/libicu55_55.1-7_amd64.deb) instead and install using “dpkg -i”.)
```
Comment (TravisEz13) [inline docs/installation/linux.md]: you must have a blank line before after a code block to be valid markdown.

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
Add a blank line after the new code block ending you added and before the new code block opening you added.

**Comment by iSazonov** [comment]
@yeyintminthuhtut Could you please address feedback?

File: docs/installation/linux.md (hunk 1 above)
Comment (yeyintminthuhtut) [reply docs/installation/linux.md]: ok indeed. thanks

**Comment by TravisEz13** [comment]
Please remove WIP from the title when the PR is read for review.

**Comment by iSazonov** [comment]
@yeyintminthuhtut Could you please continue?

**Comment by TravisEz13** [comment]
This PR should be submitted to the `PowerShell-Docs` Repo

**Pull request closed without merging**